plugins/cmd/chainlink-median/main.go

```diff
@@ -26,28 +27,45 @@ func main() {
 	defer closeLggr()
 	slggr := logger.Sugared(lggr)
 
+	telem := loop.SetupTelemetry(nil)
```

Comment:
maybe a comment here about why we don't need or want a non-nil arg?

Reply:
Updated